db/schema.rb

```diff
@@ -79,6 +72,28 @@
   add_index "pillars", ["minion_id"], name: "index_pillars_on_minion_id", using: :btree
   add_index "pillars", ["pillar"], name: "index_pillars_on_pillar", using: :btree
 
+  create_table "registries", force: :cascade do |t|
+    t.string   "name",                   limit: 255
+    t.string   "url",                    limit: 255
+    t.integer  "certificate_service_id", limit: 4
```

Comment:
This should be removed, the proxy model will take this into account by linking `certificate_id` with a `service_type` and a `service_type` (in this case `Registry` as type and `registries`.`id` as id).